src/i18n/dim_en.json

```diff
@@ -586,7 +586,9 @@
       "TranslationText": "We use {{link}} for ease of translation. If you want to improve one of DIM's translations, join the team.",
       "Twitter": "Twitter",
       "TwitterHelp": "Be sure to follow us to stay on top of updates and realtime application state information.",
-      "Version": "Version {{version}} ({{flavor}}), built on {{date}}"
+      "Version": "Version {{version}} ({{flavor}}), built on {{date}}",
+      "ReleaseSchedule": "This version of DIM is updated once a week, in the afternoon on Saturdays, US Pacific time.",
```

Comment:
Also move them to the alphabetical location or run `grunt download_translation` or register another task that just sorts dim_en.json for PRs towards dim_en.json.